requirements.txt

```diff
@@ -1,9 +1,11 @@
+-e .
```

Comment:
Currently, the Git action workflow is configured for Python versions 3.9 to 3.11, while I have been working with Python 3.12 😅. This discrepancy makes it a bit challenging to identify compatible packages, and the process might take longer than expected. However, since the library is set up with tox for managing different package versions, I’ll be postponing this for now 😁.